base/applications/iexplore/iexplore.rc

```diff
@@ -27,7 +27,7 @@
 #include "wine/wine_common_ver.rc"
 
 /* @makedep: iexplore.ico */
-1 ICON iexplore.ico
+1 ICON "iexplore.ico"
 
 /* @makedep: iexplore.inf */
-REGINST REGINST iexplore.inf
+REGINST REGINST "iexplore.inf"
```

Comment:
Does adding the quotations fix anything specific?

Reply:
Reverted.